### ISSUE TITLE
Error acquiring position in Firefox

### ISSUE THREAD

**Issue by critzo** [throneless-tech/piecewise-saas#16]
![image](https://user-images.githubusercontent.com/4633345/92612503-ee418700-f287-11ea-9b46-ad2637f857a8.png)

**Comment by jheretic** [comment]
I'll see if I can track that down, that error message seems to come from the browser rather than our code directly. Few quick questions:

* I assume you consented to the location lookup? (just checking, it would be a different kind of error if you didn't consent and it still attempted the lookup)
* Is your browser set to allow sites to request location access, and did you permit the location lookup when the request was made?
* Are you running any extensions that might block lookups to either the Google location API or Mozilla location API?

**Comment by critzo** [comment]
Yes, consented to location lookup, my browser is set to allow sites to request location, and I did permit the request. No extensions that I'm aware of that would block the Google or Mozilla API
